js/create_project_progress.jsx

```diff
@@ -0,0 +1,83 @@
+var DeployKeyTest = require('./deploy_key_test.jsx');
+
+var CreateProjectProgress = React.createClass({
+
+	checkInterval: false,
+
+	getInitialState: function() {
+		return {
+			complete: false,
+			checkingStatus: false
+		};
+	},
+
+	componentDidMount: function() {
+		this.checkInterval = window.setInterval(this.handleCheckStatus, 10000);
```

Comment:
This is cleaned up on line 34: https://github.com/silverstripe/deploynaut/pull/427/files#diff-c5e6c9222980acfeb9e5ecc3f982755dR34